docs/features/rgb_matrix.md

```diff
@@ -589,13 +589,13 @@ This function can only be run from within an effect or indicator callback, other
 
 #### Arguments {#api-rgb-matrix-set-color-arguments}
 
- - `uint8_t index`  
+ - `uint8_t index`<br />
```

Comment:
If you're going to do this, you'll have to update every other page where I've added these API reference sections. I'd rather you just leave these as is for now.

Maybe the argument lists should just look like

- `uint8_t r`: The red value to set.
- `uint8_t g`: The green value to set.
- `uint8_t b`: The blue value to set.

So they don't have to rely on trailing whitespace.